app/views/va_facilities/_facility_status_definitions_modal.html.erb

```diff
@@ -1,7 +1,13 @@
-<a href="#facility-status-modal" aria-controls="facility-status-modal" data-open-modal class="fas fa-question-circle cursor-pointer">
+<%
+  require 'securerandom'
+  guid = SecureRandom.hex(10) #or whatever value you want instead of 10
+%>
+
+<div href="#facility-status-modal-<%= guid %>" aria-controls="facility-status-modal-<%= guid %>" data-open-modal class="cursor-pointer">
```

Comment:
What is the reason for adding this to the `id`? 🤔

Reply:
Was getting accessibility (wcag 508)... failure due to duplicate ids on the same page so this is to ensure the modal ids are unique.

Reply:
Should we have a duplicate of this modal though? Shouldn't we be loading this modal once anyways?

Reply:
I think it gets called in a loop... so there is multiple entries.. this is inside the practice editor Adoption "Status" label/link.  Gets called in a loop..  lets do a sanity check tomorrow..